app/scripts/views/settings/recovery_codes.js

```diff
@@ -0,0 +1,142 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const $ = require('jquery');
+const Cocktail = require('cocktail');
+const FormView = require('../form');
+const ModalSettingsPanelMixin = require('../mixins/modal-settings-panel-mixin');
+const Template = require('templates/settings/recovery_codes.mustache');
+const RecoveryCodePrintTemplate = require('templates/settings/recovery_codes_print.mustache');
+const RecoveryCode = require('../../models/recovery-code');
+
+var t = FormView.t;
+
+const View = FormView.extend({
+  template: Template,
+  className: 'recovery-codes',
+  viewName: 'settings.two-step-authentication.recovery-codes',
+
+  events: {
+    'click .copy-codes': FormView.preventDefaultThen('_copyCodes'),
```

Comment:
nit: Can you set up a reference to `preventDefaultThen` to save a few bytes?

```js
const {preventDefaultThen} = FormView;
```
You could combine that with `t` above:
```js
const {preventDefaultThen, t} = FormView;
```